packages/mui-material/src/Select/SelectInput.js

```diff
@@ -273,7 +302,10 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
 
         Object.defineProperty(clonedEvent, 'target', {
           writable: true,
-          value: { value: newValue, name },
+          value: {
+            value: newValue,
+            name,
+          },
```

Comment:
Please revert this change (and others below) as it's not related to the scope of this PR.

Reply:
Reverted.